f8a_worker/models.py

```diff
@@ -428,3 +428,18 @@ class OSIORegisteredRepos(Base):
     git_sha = Column(String(255), nullable=False)
     email_ids = Column(String(255), nullable=False)
     last_scanned_at = Column(DateTime)
+
+
+class ComponentAnalysesRequests(Base):
+    """Table for CA request."""
+
+    __tablename__ = "component_analyses_requests"
+
+    component_analyses_request_id = Column(String(64), primary_key=True)
+    request_id = Column(String(64), nullable=True)
```

Comment:
This is should be a UUID. It will be generated for each request.

Reply:
Done